### PR TITLE
Task_01_Standard_Library

### PR DESCRIPTION
**Pull request by KatsiarynaShymko**

### CODE REVIEW

**Comment by akhmialeuski** [comment]
What’s good:
- Correctly uses `json`, `datetime`, `os.path.exists`.
- Age is calculated accurately considering the day and month (edge case handled).

Problems and risks:
- **Lack of functions.** Logic is harder to test; no docstrings or type hints.
- **Lack of validation.** No checks for keys, date format, or value types; potential crashes on corrupted records.
- **Output.** Raw dict is printed; better to structure the output.

Recommendations:
- Break into functions with type annotations and reST-style docstrings (Pythonic, testable).
- Add error handling (`JSONDecodeError`, `ValueError`) and skip invalid records with logging.
- Format the output results.
- I recommend moving the main logic into a separate `main()` function, and wrapping the program execution with `if __name__ == "__main__": main()`. This will make the code cleaner, facilitate reusing functions, and simplify testing.

**Comment by KatsiarynaShymko** [comment]
Code is updated based on provided comments

File: Task_Standard_Library.py [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+            print("There are issues with JSON file")
+        else:
+            new_list = []
+            for person in data:
+                dob = person.get("birth_date", "")
+                person_age = age_calculation(dob)
+                if person_age is not None and person_age < 18:
+                    new_list.append(person)
+            for row in new_list:
+                print(json.dumps(row, ensure_ascii=False, indent=2))
```
Comment (akhmialeuski) [inline Task_Standard_Library.py:49]: I meant the following

```python
# Print table header
            print(f"{'Name':<20} {'Birth Date':<12} {'Gender':<8} {'Age':<4}")
            print("-" * 48)
            
            # Print table rows
            for person in new_list:
                name = person.get("name", "")
                birth_date = person.get("birth_date", "")
                gender = person.get("gender", "")
                age = age_calculation(birth_date)
                print(f"{name:<20} {birth_date:<12} {gender:<8} {age:<4}")
```
with output
```bash
ame                 Birth Date   Gender   Age 
------------------------------------------------
Jane Doe             2010-02-01   Female   15  
Elis Grey            2012-12-11   Female   12  
Petro San            2010-02-12   Female   15  
Vasi Pen             2011-06-05   Male     14  
Van Dame             2013-07-07   Male     12  
VASILE Doe           2009-11-11   Male     15  
PETRE Doe            2018-04-11   Female   7 
```
But this is not part of the current task, so just keep it in mind for the future, as it’s much more convenient to say it that way.
